lndmanage/lib/rating.py

```diff
@@ -52,13 +53,16 @@ def node_to_node_weight(self, u, v, e, amt_msat):
         ]
         return min(costs)
 
-    def channel_weight(self, node_from: str, node_to: str, channel_info: Dict, amt_msat):
+    def channel_weight(self, node_from: NodeID, node_to: NodeID, channel_info: Dict, amt_msat: int):
+
         # check if channel is blacklisted
         if self.blacklisted_channels.get(channel_info["channel_id"]) == {"source": node_from, "target": node_to}:
             return math.inf
 
+        capacity = self.network.max_pair_capacity[channel_info['keypair']]
+
         # we don't send if the channel cannot carry the payment
-        if amt_msat // 1000 > channel_info["capacity"]:
+        if amt_msat // 1000 > capacity:
```

Comment:
This can miss cases where fractional sats still make it over. 

```
if amt_msat > capacity * 1000:
```

Reply:
right, thanks!